code/API_definitions/device-status.yaml

```diff
@@ -266,13 +773,54 @@ components:
             status: 503
             code: "UNAVAILABLE"
             message: "Service unavailable"
-    Generic504:
-      description: Request time exceeded. If it happens repeatedly, consider reducing the request complexity
-      content:
-        application/json:
-          schema:
-            $ref: "#/components/schemas/ErrorInfo"
-          example:
-            status: 504
-            code: "TIMEOUT"
-            message: "Request timeout exceeded. Try later."
+  examples:
+    ROAMING_STATUS:
+      value:
+        subscriptionId: 123654
```

Comment:
Hi! I'm new here so please ignore this if it causes any noise :D

Shouldn't the property in the examples also be `eventSubscriptionId`? It happens in all examples

Reply:
Thanks @fernandopradocabrillo 

@bigludo7 
Can you update the notification examples?

Reply:
Hi @eric-murray,
@bigludo7 is OoO, I made PR #64 to fix typos

BR,
Patrice

Reply:
Thanks @patrice-conil. I merged these changes into the PR, but that invalidates previous approvals.

@sachinvodafone, @akoshunyadi, @JoachimDahlgren Can you review again?

@fernandopradocabrillo Can you review and approve for Telefonica?